src/checkbox/checkbox.styles.ts

```diff
@@ -60,6 +61,8 @@ export const checkboxStyles = (
 	.checked-indicator {
 		width: 100%;
 		height: 100%;
+		min-width: ${typeRampPlus1FontSize};
+		min-height: ${typeRampPlus1FontSize};
```

Comment:
Instead of `typeRampPlus1FontSize` could we use `designUnit`?

Something like this: `calc(${designUnit} * 4px);`

Reply:
Sounds good! Thanks @hawkticehurst